File: erpadda/projects/report/project_billing_summary/project_billing_summary.py
# Copyright (c) 2013, VMRaid and contributors
# For license information, please see license.txt


import vmraid

from erpadda.projects.report.billing_summary import get_columns, get_data


def execute(filters=None):
	filters = vmraid._dict(filters or {})
	columns = get_columns()

	data = get_data(filters)
	return columns, data

File: erpadda/projects/report/project_billing_summary/project_billing_summary.js
// Copyright (c) 2016, VMRaid and contributors
// For license information, please see license.txt
/* eslint-disable */

vmraid.query_reports["Project Billing Summary"] = {
	"filters": [
		{
			fieldname: "project",
			label: __("Project"),
			fieldtype: "Link",
			options: "Project",
			reqd: 1
		},
		{
			fieldname:"from_date",
			label: __("From Date"),
			fieldtype: "Date",
			default: vmraid.datetime.add_months(vmraid.datetime.month_start(), -1),
			reqd: 1
		},
		{
			fieldname:"to_date",
			label: __("To Date"),
			fieldtype: "Date",
			default: vmraid.datetime.add_days(vmraid.datetime.month_start(),-1),
			reqd: 1
		},
	]
}
